homeassistant/components/zwave/__init__.py

```diff
@@ -313,30 +314,20 @@ def _add_node_to_component():
             _add_node_to_component()
             return
 
-        async def _check_node_ready():
-            """Wait for node to be parsed."""
-            start_time = dt_util.utcnow()
-            while True:
-                waited = int((dt_util.utcnow()-start_time).total_seconds())
-
-                if entity.unique_id:
-                    _LOGGER.info("Z-Wave node %d ready after %d seconds",
-                                 entity.node_id, waited)
-                    break
-                elif waited >= const.NODE_READY_WAIT_SECS:
-                    # Wait up to NODE_READY_WAIT_SECS seconds for the Z-Wave
-                    # node to be ready.
-                    _LOGGER.warning(
-                        "Z-Wave node %d not ready after %d seconds, "
-                        "continuing anyway",
-                        entity.node_id, waited)
-                    break
-                else:
-                    await asyncio.sleep(1, loop=hass.loop)
+        def _on_ready(sec):
+            _LOGGER.info("Z-Wave node %d ready after %d seconds",
+                         entity.node_id, sec)
+            hass.async_add_job(_add_node_to_component)
 
+        def _on_timeout(sec):
```

Comment:
Can you tag this one and the other async callbacks with `@callback` so we know it's async

Reply:
Done